db/db_bench.cc

```diff
@@ -926,23 +900,9 @@ struct DBWithColumnFamilies {
   DBWithColumnFamilies(const DBWithColumnFamilies& other)
       : cfh(other.cfh),
         db(other.db),
-        txn_db(other.txn_db),
         num_created(other.num_created.load()),
         num_hot(other.num_hot) {}
 
-  void DeleteDBs() {
-    std::for_each(cfh.begin(), cfh.end(),
-                  [](ColumnFamilyHandle* cfhi) { delete cfhi; });
-    cfh.clear();
-    if (txn_db) {
-      delete txn_db;
-      txn_db = nullptr;
-    } else {
-      delete db;
-    }
-    db = nullptr;
-  }
-
```

Comment:
I noticed quite some functions were deleted. Are they repository sync and merge issue?